src/libs/OptionsListUtils.js

```diff
@@ -252,8 +253,7 @@ function createOption(logins, personalDetails, report, {
     let text;
     let alternateText;
     if (isChatRoom || isPolicyExpenseChat) {
-        text = lodashGet(report, 'reportName')
-            || lodashGet(report, 'oldPolicyName', '');
+        text = ReportUtils.getReportName(report, personalDetailMap, policies);
```

Comment:
I think we should use `ReportUtils.getReportName` for the `text` even if the report is not a chat room or policyExpenseChat. If you look at lines 261 -> 264, it's essentially doing the same thing that `ReportUtils.getReportName` would do, right?

Reply:
Great catch, just updated this.